src/components/material/MaterialPeriodical.tsx

```diff
@@ -32,12 +34,26 @@ const MaterialPeriodical: FC<MaterialPeriodicalProps> = ({
     })
     .flat();
 
-  return (
-    <MaterialPeriodicalSelect
-      groupList={groupObjectArrayByProperty(materialsPeriodical, "volumeYear")}
-      selectPeriodicalSelect={selectPeriodicalSelect}
-    />
+  const groupByvolumeYear = groupObjectArrayByProperty(
```

Comment:
```suggestion
  const groupByVolumeYear = groupObjectArrayByProperty(
```

Proper casing makes the code a bit easier to read.